### PR TITLE
Adding 42 Born2Code FR and US

### PR DESCRIPTION
**Pull request by Tifox**
Hi adding the famous tuition free school :)

### CODE REVIEW

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -21596,6 +21603,13 @@
     {
         "alpha_two_code": "FR",
         "country": "France",
+        "domain": "student.42.fr",
+        "name": "42 FR",
+        "web_page": "http://www.42.fr//"
```
Comment (yigitguler) [inline world_universities_and_domains.json]: This has double trailing slashes. Could you please fix it? Thanks.

**Comment by yigitguler** [comment]
Sure. Will be happy to merge. However there is a minor problem.

File: world_universities_and_domains.json (hunk 1 above)
Comment (Tifox) [inline world_universities_and_domains.json]: hey, sorry, it's fixed :)

**Comment by yigitguler** [comment]
Thanks!

**Comment by Tifox** [comment]
thanks to you 👍

**Pull request merged**